### PR TITLE
Parse .spec.features.domains and pass as environment variables

### PR DESCRIPTION
**Pull request by dvaldivia**
Parses the domains specified on `.spec.features.domains` and configures accordingly `MINIO_DOMAIN`, `MINIO_SERVER_URL` and `MINIO_BROWSER_REDIRECT_URL`

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by bexsoft — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

File: pkg/resources/statefulsets/minio-statefulset.go [hunk 1]
```diff
@@ -134,24 +118,33 @@ func minioEnvironmentVars(t *miniov2.Tenant, wsSecret *v1.Secret, hostsTemplate
...
+
 	// Set the redirect url for console
 	if t.HasConsoleDomains() {
-		schema := "http"
-		if t.TLS() {
-			schema = "https"
-		}
 		envVars = append(envVars, corev1.EnvVar{
 			Name:  "MINIO_BROWSER_REDIRECT_URL",
-			Value: fmt.Sprintf("%s://%s", schema, t.Spec.Features.Domains.Console),
```
Comment (harshavardhana) [inline pkg/resources/statefulsets/minio-statefulset.go:154]: isn't this a breaking change?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: pkg/resources/statefulsets/minio-statefulset.go (hunk 1 above)
Comment (dvaldivia) [reply pkg/resources/statefulsets/minio-statefulset.go:154]: A little, I'm hoping this change is not too disruptive, alternatively I could attach the scheme if it's not present.

Comment (harshavardhana) [reply pkg/resources/statefulsets/minio-statefulset.go:154]: That is needed IMO, we can't break existing deployments.

Comment (dvaldivia) [reply pkg/resources/statefulsets/minio-statefulset.go:154]: addressed, made it so if the schema is missing I add it, just like before

**Review by harshavardhana — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**